be/src/olap/field.h

```diff
@@ -161,6 +161,19 @@ class Field {
         return _type_info->direct_copy(dst->mutable_cell_ptr(), src.cell_ptr());
     }
 
+    // deep copy source cell' content to destination cell.
+    // For string type, this will allocate data form pool,
+    // and copy srouce's conetent.
+    template<typename DstCellType, typename SrcCellType>
+    void copy_object(DstCellType* dst, const SrcCellType& src, MemPool* pool) const {
```

Comment:
If the template is not necessary, we can implement this in this file other than change TypeInfo.